README.md

```diff
@@ -126,4 +126,91 @@ In order for our scripts to work, the corresponding password for the `ega-box-12
-Once you have correct permissions, you can navigate to the `sc-ega-submission` [Secret Manager](https://console.cloud.google.com/security/secret-manager?referrer=search&authuser=0&project=sc-ega-submissions) console page, click "Create Secret" at the top of the page, and follow the prompts for storing a new password. Once the new password is saved, you can then follow the directions outlined above for submitting your Terra workflows. 
+Once you have correct permissions, you can navigate to the `sc-ega-submission` [Secret Manager](https://console.cloud.google.com/security/secret-manager?referrer=search&authuser=0&project=sc-ega-submissions) console page, click "Create Secret" at the top of the page, and follow the prompts for storing a new password. Once the new password is saved, you can then follow the directions outlined above for submitting your Terra workflows.
+
+## Field Mappings
+- **aggregation_path**: Pulled from on-prem servers for bams, and Clio for crams.
+- **analysis_type**: Pulled from both `picard.read_group_index` and `picard.rapid_qc_index`.
+- **avg_mean_insert_size**: Takes the average of `MEAN_INSERT_SIZE` field from the `metrics.aggregation_insert_size` table.
+- **avg_standard_deviation**: Takes the average of `STANDARD_DEVIATION` field from the `metrics.aggregation_insert_size` table.
+- **group_library_name**: Takes each distinct `library_name` from the query and concatenates each with "_".
+- **illumina_instrument**: `illumina_instrument` field from the `slxasync.runs` table.
+- **library_layout**: Uses the field `PAIRED_RUN` from the index tables. Returns 'single' when == 0 else returns 'paired'.
+- **library_selection**:
+  - Case 1: if `index.analysis_type` = 'Resequencing' AND `index.library_type` = 'WholeGenomeShotgun' THEN 'RANDOM'.
+  - Case 2: 'cDNA' if `index.analysis_type` = 'cDNA'.
+  - Case 3: 'Hybrid Selection' if `index.library_type` = 'HybridSelection'.
+  - Case 4: 'Random' if `index.library_type` = 'WholeGenomeShotgun'.
+  - Case 5: 'OTHER' if the above conditions are not met.
+- **library_source**:
+  - Case 1: 'GENOMIC' if `sample_material_type` equals any of these values ('Genomic DNA', 'DNA:DNA Genomic', 'DNA:DNA Somatic', 'DNA:DNA WGA Qiagen').
+  - Case 2: 'TRANSCRIPTOMIC' if `sample_material_type` equals any of these values ('RNA:Total RNA', 'RNA').
+  - Case 3: 'VIRAL RNA' if `sample_material_type` equals 'RNA:ss Viral RNA'.
+  - Case 4: 'OTHER' if the above conditions are not met.
+- **library_strategy**:
+  - Case 1: 'WGS' if `library_type` = 'WholeGenomeShotgun'.
+  - Case 2: 'RNA-Seq' if `library_type` equals any of these values ('cDNAShotgunStrandAgnostic', 'cDNAShotgunReadTwoSense').
+  - Case 3: 'WXS' if `library_type` = 'HybridSelection'.
+  - Case 4: 'OTHER' if the above conditions are not met.
+- **library_type**: Pulled from the `library_type` field from the index tables.
```

Comment:
Should this specifically be `picard.read_group_index` or `picard.rapid_qc_index`?

Reply:
Oh right, I meant just specifically provide the table names, so similar with how you did it above with "Pulled from both `picard.read_group_index` and `picard.rapid_qc_index`."